META-INF/plugin.xml

```diff
@@ -242,6 +242,13 @@
       <add-to-group group-id="CodeFormatGroup" anchor="last"/>
       <keyboard-shortcut keymap="$default" first-keystroke="shift ctrl alt E"/>
     </action>
+    <action id="HaskForce.HindentFormat"
+            class="com.haskforce.actions.HaskellHindentFormatAction"
+            text="Reformat code with Hindent"
+            description="Format current file with Hindent">
+      <add-to-group group-id="CodeFormatGroup" anchor="last"/>
+      <keyboard-shortcut keymap="$default" first-keystroke="shift ctrl alt D"/>
```

Comment:
I don't know if you have a convention for keyboard shortcuts. I used 'D' just because it didn't seem to be in use already.